deps.repos

```diff
@@ -0,0 +1,5 @@
+repositories:
```

Comment:
This dependency shouldn't be in the `package_set`?

Reply:
It is properly set in BIR's private environment. This is a public repository, so I'm relying on tools commonly used by the ROS community. In this case, I'm providing a [vcstool](https://github.com/dirk-thomas/vcstool) description to be used by the [action-ros-ci](https://github.com/ros-tooling/action-ros-ci) to get not-released dependencies source-code (in this case, the `sonar_msgs` package).